lib/modules/apostrophe-attachments/lib/api.js

```diff
@@ -166,6 +165,9 @@ module.exports = function(self, options) {
       if (!info) {
         return callback('notfound');
       }
+      if (!self.croppable[info.extension]) {
+        return callback(new Error('uncroppable'));
```

Comment:
This won't be displayed because we head them off before they get this far in the UI. The error is in case someone is sneaky or overriding things to an insane level (: